src/pages/UniversalSwap/SwapV3/index.tsx

```diff
@@ -426,32 +450,12 @@ const SwapComponent: React.FC<{
                 searchTokenName={searchTokenName}
               />
             )}
+
+            <div className={cx('ratio')}>
+              {`1 ${originalFromToken?.name} ≈ ${averageRatio?.displayAmount || '0'} ${originalToToken?.name}`}
```

Comment:
i think originalFromToken & originalToToken alway available, so we dont need to use optional chaining in this line :v

Reply:
with averageRatio, because it is used many time, i think we can init default value for it with displayAmount = 0, so we dont need to check displayAmount & logical || anymore. What do you think?

Reply:
oki